src/cart/crt_iv.c

```diff
@@ -3135,7 +3135,7 @@ crt_iv_update_inline_hdlr(crt_rpc_t *rpc, struct crt_ivns_internal *ivns_interna
 decref_error:
 	IVNS_DECREF(ivns_internal);
 put_error:
-	iv_ops->ivo_on_put(ivns_internal, &iv_value, &user_priv);
+	iv_ops->ivo_on_put(ivns_internal, &iv_value, user_priv);
```

Comment:
unrelated to your patch, but i don't understand why on get is **arg, but on_put is *arg

Reply:
on_get() populates the pointer passed (**arg). (alternatively on_get() could have returned ptr, instead of int rc, not requiring **arg then).

on_put() indicates to the iv implementation to free whatever was allocated by on_get(), so just needs a pointer value (so just *arg)